docs/source/commands/peerchannel.md

```diff
@@ -31,7 +31,7 @@ Usage:
   peer channel [command]
 
 Available Commands:
-  create               Create a channel
+  create               deprecated: Create a channel
```

Comment:
```suggestion
  create               [DEPRECATED] Create a channel
```
I think it would be better to use the same expression for deprecated as for other commands.
https://hyperledger-fabric.readthedocs.io/en/latest/commands/configtxgen.html